devel/glasskube/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/glasskube/glasskube 0.17.0 v
+go.offline_build    no
+github.tarball_from archive
+revision            0
+
+homepage            https://glasskube.dev/
+
+categories          devel
+installs_libs       no
+maintainers         {esafak @esafak} openmaintainer
```

Comment:
this implies that you have an email address is esafak@macports.org - is that the case? If not, please add your obfuscated valid mail address.

Reply:
Should be fixed